### PR TITLE
Filled in Docs for Panorama Sky

### PR DESCRIPTION
**Pull request by clayjohn**
Just trying to get some work done completing the class ref!

### CODE REVIEW

File: doc/classes/PanoramaSky.xml [hunk 1]
```diff
@@ -1,8 +1,10 @@
 <?xml version="1.0" encoding="UTF-8" ?>
 <class name="PanoramaSky" inherits="Sky" category="Core" version="3.1">
 	<brief_description>
+		A type of [Sky] used to draw a background texture.
 	</brief_description>
 	<description>
+		A resource referenced in an [Environment] that is used to draw a background. The Panorama sky functions similar to skyboxs in other engines except it uses a equirectangular sky map instead of a cube map.
```
Comment (mhilbrunner) [inline doc/classes/PanoramaSky.xml]: skyboxes

**Comment by clayjohn** [comment]
Should be fixed now!

**Comment by mhilbrunner** [comment]
Looks good. Thanks! :)

**Comment by hpvb** [comment]
Cherry picked into 3.0.3

**Pull request merged**